internal/logger/target/http/http.go

```diff
@@ -369,32 +496,43 @@ func New(config Config) *Target {
 		ctransport.Proxy = http.ProxyURL(proxyURL)
 		h.config.Transport = ctransport
 	}
-	h.client = &http.Client{Transport: h.config.Transport}
 
-	return h
+	ctransport := h.config.Transport.(*http.Transport).Clone()
+	ctransport.TLSClientConfig.InsecureSkipVerify = true
```

Comment:
`InsecureSkipVerify` shouldn't be the default behavior.

Reply:
yep sorry! this should have been removed

Reply:
Removed